eframe/src/lib.rs

```diff
@@ -95,9 +100,44 @@ pub use web_sys;
 ///     eframe::start_web(canvas_id, Box::new(|cc| Box::new(MyEguiApp::new(cc))))
 /// }
 /// ```
+#[cfg(target_arch = "wasm32")]
+use arc_swap::ArcSwapOption;
+
+#[cfg(target_arch = "wasm32")]
+unsafe impl Send for AppRunner {}
+
+#[cfg(target_arch = "wasm32")]
+unsafe impl Sync for AppRunner {}
+
+#[cfg(target_arch = "wasm32")]
+static GLOBAL_HANDLE: ArcSwapOption<Mutex<AppRunner>> = ArcSwapOption::const_empty();
```

Comment:
Adding a dependency on `ArcSwapOption` seems a bit unnecessary to me. Perhaps `Mutex<Option<AppRunnerRef>>` could do?

Reply:
I dont know how to do it. I have `Arc<Mutex<AppRunnerRef>>` how I can  place it in static variable?

Reply:
Good option could be return app handle to js, but singleton is good too, for me

Reply:
Returning it to JS is much nicer actually, since we can then also support multiple egui:s from the same WASM.